lib/src/components/NxGlobalSidebar/NxGlobalSidebar.scss

```diff
@@ -115,26 +133,36 @@ $-separator-color: var(--nx-swatch-indigo-40);
     margin: 0;
   }
 
-  &:focus {
+  &:focus:not(.selected) {
     border-radius: 0;
-    outline: 2px solid $-focus-color;
-    outline-offset: -2px;
+    outline: var(--nx-focus-outline-width) solid var(--nx-color-global-sidebar-menu-icon-border-focus);
```

Comment:
I thought `--nx-color-global-sidebar-menu-icon...` was for the toggle button?

Reply:
> I thought `--nx-color-global-sidebar-menu-icon...` was for the toggle button?

I changed it to be more general: 6dd4269192bb7db67f9a7dbe15fc63d8d72f9a0e